Automattic/jetpack
# VideoPress: Add upload progress to VideoThumbnail and thumbnail states on VideoRow

Fixes #27085

#### Changes proposed in this Pull Request:

* Adds upload progress percentage to the video thumbnail
* Adds the uploading and processing states to the video thumbnail on video row
* Moves the thumbnail logic to the VideoThumbnail component, so changes are applied to both card and row views

#### Other information:


#### Jetpack product discussion


N/A

#### Does this pull request change what data or activity we track or use?


No.

#### Testing instructions:






* Go to the VideoPress dashboard and upload a video while on the grid view
* Check that the upload percentage is shown
* Repeat the process, now on the list view
* Check that the uploading and processing states now are shown

Grid view:

https://user-images.githubusercontent.com/8486249/198131750-1f206315-3fd6-48f4-9fe0-85190c46eb36.mp4

List view:

https://user-images.githubusercontent.com/8486249/198131743-85849b0b-40bc-40cb-8c7e-45398f1ef124.mp4
